packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -229,7 +229,7 @@ async function getAuthHandler(module) {
       version = version.split('+')[0]
     }
 
-    const versionIsPublished = JSON.parse(stdout).versions.includes(version)
+    const versionIsPublished = stdout.includes(version)
```

Comment:
I am having trouble replicating this error where `stdout` fails to parse to JSON.

I can see how it could be invalid JSON as we never validate what comes back from the command. We assume if it completed successfully (execa didn't throw an error) then we got valid JSON which could be wrong.

Could you provide some instructions on how I can replicate this error? Did you manage to capture what value `stdout` had when it failed? I think you said it was an array.

Reply:
I'll post my results tomorrow as I need to re-run this in a separate redwood repository. I was able to get unblocked by making this change.

When I logged into console, I noticed something resembling this:
```log
console.log(stdout);

[
   ...
   "6.3.2",
   "8.5.1+...",
   ...
]
```
`stdout` was an array-type that was not able to be parsed by `JSON.parse`.

Reply:
Okay thanks for your reply!